contrib/cni/README.md

```diff
@@ -11,6 +11,54 @@ specified by `crio.network.network_dir` in your `crio.conf`).
 In addition, you need to install the [CNI plugins][cni] necessary into
 `/opt/cni/bin` (or the directories specified by `crio.network.plugin_dir`). The
 two plugins necessary for the example CNI configurations are `loopback` and
-`bridge`.
+`bridge`. Below is a tutorial on downloading and setting up the [CNI plugins][cni]
 
 [cni]: https://github.com/containernetworking/plugins
+
+### Plugins tutorial
+
+This tutorial will use the latest version of `CNI` plugins from the master branch and build it from source.
+
+Download the `CNI` plugins source tree:
+
+```
+git clone https://github.com/containernetworking/plugins
+```
+
+```
+cd plugins
+```
+
+Build the `CNI` plugins:
+
+```
+./build_linux.sh # or build_windows.sh
+```
+
+Output:
+
+```
+Building API
+Building reference CLI
+Building plugins
+   flannel
+   tuning
+   bridge
+   ipvlan
+   loopback
+   macvlan
+   ptp
+   dhcp
+   host-local
+   noop
+```
+
+Install the `CNI` plugins:
+
+```
+sudo mkdir -p /opt/cni/bin
+```
+
+```
+sudo cp bin/* /opt/cni/bin/
+```
```

Comment:
I'd just make this one code block, drop lines 60 and 62.  Keep 61 if it looks good with it.